Input.py

```diff
@@ -14,7 +14,7 @@ def getButton(self, ButtonName : str, XboxController : wpilib.XboxController):
         Takes in a string which acts as the key for a button and returns whether or not the button is pressed.
         """
         buttons = {"XButton" : XboxController.getXButtonPressed(),
-        "AButton" : XboxController.getAButton(),"BButton" : XboxController.getBButtonPressed(),
+        "AButton" : XboxController.getAButton(),"BButton" : XboxController.getBButton(),
```

Comment:
style: If this change is necessary, it should be made consistently with the other `get[button]Pressed` functions.